src/dialogs/DontAskAgainDialog/DontAskAgainDialog.js

```diff
@@ -111,4 +119,6 @@ DontAskAgainDialog.defaultProps = {
     confirm: 'Confirm',
     checkbox: 'Do not ask again',
   },
+  confirmButtonProps: { color: 'primary' },
+  cancelButtonProps: { color: 'primary' },
```

Comment:
Can we keep the default color defined "inline" in the component instantiation ?

This way, it should still be overridden if color is defined in `confirmButtonProps` (or `cancelButtonProps`), but it would prevent losing the value of `color` if a dev provides an object that doesn't define `color` (e.g. a dev wants to provide only `variant` in `confirmButtonProps`)

If this works, then we probably don't need to define a `defaultProps` for these custom button props

Reply:
DONE